open-company/open-company-web
# FoC gaps

Card: https://trello.com/c/JZn0WBWp

To test:
- check gaps btw expanded posts
- check gaps btw expanded posts and date separators
- check no gaps in collapsed-mode
- check in collasped mode the date separator still have the right gaps
- check mobile gaps
